index.html

```diff
@@ -12,50 +12,67 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <header>
 
-  <nav class="nav-bar">
-    <section class="logo"> 
+  <div class="nav-bar">
+
+    <div class="logo"> 
          <a href="#"><img src="./images/My Logo.png" alt="logo for the website"></a> 
-        </section>
+    </div>
   
-        <section class="dashboard">  
+        <div class="dashboard">  
```

Comment:
- [ ] The previous reviewer requested that you make use of the semantic `<nav>` tag to wrap the hamburger icon. please you are to make this correction because it is best practices to use semantic tags and not non-semantic tags like `<div>`, it is good to follow proper HTML standard and it helps with search engine optimization (SEO) :+1: